server/api/routers/tasks.ts

```
@@ -268,121 +300,69 @@ export const tasksRouter = router({
-        const task = await ctx.prisma.task.findUnique({
-          where: { id: input.id },
-          select: { teamId: true },
-        });
-
-        if (!task || !task.teamId) {
-          throw new TRPCError({
-            code: "NOT_FOUND",
-            message: "Task not found",
-          });
+        const updatePayload: any = { ...updateDataInput };
+        if (input.hasOwnProperty("deadline")) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)